modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,97 @@ definition {
 		}
 	}
 
+	@description = "LPS-173894 - Verify that after updating a system object entry, triggered email notifications using Object Author and Current User terms are filled correctly for the To, From Address, From Name, CC, BCC and Template Body fields"
+	@priority = 5
+	test CanUseObjectAuthorAndCurrentUserTermsInEmailNotificationOnSystemObject {
+		property custom.properties = "feature.flag.LPS-173894=true";
+
+		task ("Given: A connected email server and the notification interval set to one minute") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "System Settings");
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Notifications",
+				configurationName = "Notification Queue",
+				configurationScope = "System Scope");
+
+			SystemSettings.editTextSetting(
+				settingName = "Notification Queue Entry Delete Interval",
+				settingValue = 1);
+
+			SystemSettings.saveConfiguration();
+		}
+
+		task ("And given: An email notification template with the applicable fields filled with the Author and Current User terms") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "[%USER_EMAILADDRESS%]",
```

Comment:
This terms that you're using are not the right terms from the story. Make sure to use the _CURRENT_USER_FIELD_ and _OBJECTNAME_AUTHOR_FIELD_.